src/styles/main-file.scss

```diff
@@ -0,0 +1,21 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&family=Roboto:wght@400;500;700&display=swap");
```

Comment:
Consider importing Google fonts in your HTML file within a <link> tag. It's more efficient and easier to manage.